src/pages/index.tsx

```diff
@@ -49,42 +49,22 @@ function Hit(props: HitProps) {
           </div>
         </CardTitle>
         <CardDescription>
-          {category.key === 'nfts.collection' && (
-            <div>
-              <p>{trait}</p>
-              <div className="grid grid-cols-6">
-                {matchedNFTs.map((nft: any, i: number) => (
-                  <div key={i} className="py-2">
-                    <Avatar>
-                      <AvatarImage
-                        src={nft.media}
-                        alt={nft.name}
-                        className="w-12 object-cover"
-                      ></AvatarImage>
-                    </Avatar>
-                  </div>
-                ))}
-              </div>
-            </div>
-          )}
-          {category.key === 'usedChains' && (
-            <div>
-              {matchedChains.map((chain: any, i: number) => (
-                <div key={i} className="flex py-4 flex-row gap-2">
-                  <Image src={CHAIN_META[chain].logo} width={20} height={20} alt={chain} />
-                  <span>{CHAIN_META[chain].name} user</span>
-                </div>
-              ))}
-            </div>
-          )}
-
+          <div className="p-4 grid grid-cols-3 gap-4">
+            {matchedMints.map((mint: any, i: number) => (
+              <a href={toZoraUrl(mint.contractAddress, mint.tokenId)} key={i} target="_blank">
```

Comment:
for many mints, the image is the same, so this ends up cluttering the results unnecessarily, esp. as most users seem to only have 1 mint apiece?

do we display all mints for a given user even if searched for a sepcific one?

also it's a little jank when there are multiple mints of the same drop for a given user, image just repeats. might make more sense to do something like 'MINT_NAME x 3', stylized nicely ofc